aws_lambda_powertools/utilities/idempotency/persistence/base.py

```diff
@@ -196,6 +196,8 @@ def _get_hashed_idempotency_key(self, data: Dict[str, Any]) -> Optional[str]:
     def is_missing_idempotency_key(data) -> bool:
         if type(data).__name__ in ("tuple", "list", "dict"):
             return all(x is None for x in data)
```

Comment:
```suggestion
        if isinstance(data, (tuple, list, dict)):
            return all(x is None for x in data)
```

Should we change this to keep the same type of checks?